graphql-dgs-spring-webmvc-autoconfigure/src/test/kotlin/com/netflix/graphql/dgs/webmvc/autoconfigure/GraphiQLPathConfigWithCustomGraphQLPathTest.kt

```diff
@@ -27,7 +27,7 @@ import org.springframework.boot.test.web.client.TestRestTemplate
     webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT,
     properties = ["dgs.graphql.path=/zuzu"]
 )
-class GraphiQLPathConfigCustomGraphQLPathTest(@Autowired val restTemplate: TestRestTemplate) {
+class GraphiQLPathConfigWithCustomGraphQLPathTest(@Autowired val restTemplate: TestRestTemplate) {
```

Comment:
**[UNRELATED]** Renamed the GraphiQL config tests to clearly indicate the scenario being tested.